api/apps/articles/model/schema.coffee

```diff
@@ -77,6 +77,7 @@ denormalizedArtwork = (->
   thumbnail_teaser: @string().allow('', null)
   thumbnail_image: @string().allow('', null)
   tags: @array().items(@string()).default([])
+  tracking_tags: @array().items(@string()).default([])
```

Comment:
Nice! Do you mind adding tests on the API side? We have places for both #save and #query which will be useful to have.